server/src/resolvers/Message/mutation.ts

```diff
@@ -62,6 +62,18 @@ export const createMessage = mutationField('createMessage', {
       },
     });
 
+    await prisma.membership.update({
+      data: {
+        updatedAt: created.createdAt,
+      },
+      where: {
+        userId_channelId: {
+          userId,
+          channelId,
+        },
+      },
+    });
```

Comment:
I think we can combine `prisma.channel.update` and `prisma.membership.update` into a single update using Prisma's "nested writes" feature.
Reference: https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#nested-writes

Reply:
thanks for your suggestion, I'll read and try ASAP :)

Reply:
Updated!